system-tests/e2e-suite/src/test/scala/com/hotels/styx/proxy/OriginCancellationMetrics.scala

```diff
@@ -67,7 +67,15 @@ class OriginCancellationMetrics extends FunSpec
       styxServer.metricsSnapshot.count("origins.app-1.requests.cancelled").get should be(0)
     }
 
-    it("Is incremented on an origin error") {
+
+    ignore("Is incremented on an origin error") {
```

Comment:
This ignored test will be to be changed into a functioning test before the PR can be approved.

Reply:
I will move the assertion into ChunkedDownloadSpec.